### PR TITLE
Feat creator register page

### PR DESCRIPTION
**Pull request by Emnetdegafe**
- Cofounder  register page
- Cofounder login page ready
- Reset your password on login

### CODE REVIEW

**Review by DannyS-stack — APPROVED** [review]
LGTM!

Only feedback I have is that you have a lot of styling rules in your components, would maybe be nicer to have it in a separate file. I don't think it it important enough to change it so looks good enough.

File: src/components/Cofounder/CofounderRegisteration.js [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+            shouldShowError: false,
+            validated: true,
+        },
+        role: {
+            value: '',
+            shouldShowError: false,
+            validated: false,
+        }
+    });
+    console.log('formData', formData)
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderRegisteration.js:61]: delete console.log

File: src/components/Cofounder/CofounderRegisteration.js [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+        },
+        country: {
+            validator: (val) => (val !== null),
+            shouldShowError: (val) => true,
+        },
+        role: {
+            validator: (val) => (val !== ''),
+            shouldShowError: (val) => true,
+        }
+    };
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderRegisteration.js:93]: It seems like this is duplicate code above? Can we not reuse the other object that we are using for the other registration?

File: src/components/Cofounder/CofounderRegisteration.js [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+        background-color: #ffffff;
+    }
+  `;
+// const RadioGroupContainer = styled(RadioGroup)({
+//     display: 'flex',
+//     flexWrap: 'nowrap',
+//     flexDirection: 'row',
+// })
+
+
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderRegisteration.js:434]: Is this all copy paste? Let's make the login form reusable instead please.

**Review by dashakrolik — CHANGES_REQUESTED** [review]

File: src/components/Cofounder/CofounderRegisteration.js (hunk 2 above)
Comment (Emnetdegafe) [reply src/components/Cofounder/CofounderRegisteration.js:93]: Not really, we needed to add role input to the form thus we needed to change it. It's a radio button that is required to be filled.

**Comment by Emnetdegafe** [comment]
Closed due to changes with LoginForm restructure and bug fix

**Review by dashakrolik — COMMENTED** [review]
good job

**Pull request closed without merging**